data/json/terrain.json

```diff
@@ -316,6 +331,7 @@
         "type" : "terrain",
         "id" : "t_sand",
         "name": "sand",
+        "description": "A large area of fine sand that could be useful in a number of ways, if it was extracted properly.",
```

Comment:
Might be nice to leave a note of *how* to extract.

Reply:
This is meant to offer a hint to the player, not necessarily be an answer for every character to immediately understand and recognize. Hint, the Construction menu uses "Extract", which is why that vocabulary is used specifically